TradeItIosTicketSDK2/TradeItNumericToggleInputCell.swift

```diff
@@ -23,13 +24,19 @@ class TradeItNumericToggleInputCell: UITableViewCell {
     func configureQuantityType(
         quantitySymbol: String?,
         quantity: NSDecimalNumber?,
-        maxDecimalPlaces: Int
+        maxDecimalPlaces: Int,
+        showToggle: Bool = false
     ) {
-        self.textField.placeholder = "Enter \(quantitySymbol ?? "")"
+        self.textField.placeholder = "Enter"
         self.quantityTypeButton.setTitle(quantitySymbol, for: .normal)
         self.textField.text = quantity?.stringValue
         self.textField.maxDecimalPlaces = maxDecimalPlaces
         self.onValueUpdated?(quantity)
+        if showToggle {
```

Comment:
I can probably clean this up a bit, but we can't ignore either of the cases because cells are reused and this function is called multiple times that may require the toggle to appear/disappear.